angular-spa-ui/js/admin-app/content-manager-module/contentManager-srv.js

```diff
@@ -21,9 +21,126 @@ module.exports = function () {
         });
         return tabs;
     };
+    
+    var setVisibility = function (defaultParams, params, config) {
 
+        if (!_.isEqual(defaultParams, params)) {
+            config.visibility = true;
+        }
+
+        return config;
+    }
+    
+    var findValueId = function (val, array) {
```

Comment:
Array.inArray() - as i know js has no such function - php has.
Array.indexOf() - not supported by IE 8